deepfence_server/model/registry.go

```diff
@@ -412,25 +405,7 @@ func ListImages(ctx context.Context, registryId string, filter reporters.FieldsF
 		var node ContainerImage
 		utils.FromMap(m.Props, &node)
 
-		// Extract tags from the "l" node
-		tagsValue, hasTags := l.Props["tags"]
-		if hasTags {
-			tags, ok := tagsValue.([]interface{})
-			if ok {
-				for _, tag := range tags {
-					if tagStr, ok := tag.(string); ok {
-						// create individual tag nodes
-						tagNode := node
-						tagNode.Tag = tagStr
-						res = append(res, tagNode)
-					}
-				}
-			}
-		}
-
-		// kludge: breaking here since we don't want to repeat images and duplicate the entries
-		// in the first iteration itself we get all the tags in an array and we use the first image and duplicate
-		break
```

Comment:
This is actually a bug. We are just using the first node from the result set "m" and updating the tag for that node. 
Also, since we don't have any way to associate result set "m" with tags in "l" , this would be incorrect. 
We would have to loop through the results in "m" and populate the result in "res".